### ISSUE TITLE
Anchor Wallet still not allowing Ledger Nano S setup

### ISSUE THREAD

**Issue by emmaivybriar** [greymass/anchor#1038]
Hi Aaron, I see you closed the issue, so I'm not sure if I need to reopen as a new issue?  I tried all your suggestions:
1. I opened a brand new Ledger Nano S chord/connector to use.
2. I ensured Ledger Live Manager was on and connected
3. I closed out all other wallets.
4. I restarted my computer.
5.  I verified all firmware is up to date. 

But the Anchor is still saying "reinitializing" endlessly. When I try to connect the EOS app within Ledger, it bounces me out to the main menue. 

Any other ideas? I really need to secure the Anchor . 

Thanks
Jay

**Comment by aaroncox** [comment]
I closed it since the support discussions like this typically happen on the forums and we only keep issues open if there's a bug we need to track and fix within Anchor. We can also continue discussion here if you'd prefer too, even if the issue is closed. 

Are you saying that when you open the EOS app on the Ledger, the EOS app on the Ledger closes and it goes back to the menu on the device? I am honestly not sure what is causing that, and sadly we aren't the developers of that software. That app can be found here:

https://github.com/LedgerHQ/app-eos

Anchor itself requires the EOS App to be running on the device before it can connect - so if I'm understanding the issue right, that's why it gets stuck on "reinitializing". It's trying to connect to the running EOS app but since it's not started, it's retrying over and over.

**Comment by emmaivybriar** [comment]
Thanks for continuing the discussion here Aaron. 

Yes, when I open the EOS app on the Ledger, the EOS app on the Ledger closes and it goes back to the menu on the device. But it ONLY does this when connected to the Anchor wallet. It works with virtually every other wallet and software I have. The Anchor is the only wallet I experience this issue with.

**Comment by aaroncox** [comment]
That's incredibly odd, I've never seen that before. I just updated my Ledger(s) to the latest firmware and latest version of the EOS App, and using Anchor 1.1.12, I'm not seeing the same issue. Few questions:

- What version of the Ledger app are you running? You mentioned upgrading the firmware, but figured I'd ask if you had the latest EOS app.
- Have you tried other EOSIO wallets with the Ledger, and do those work?

The only other thing I can think of is that the Ledger Live might also be interfering if it's connected.

**Comment by emmaivybriar** [comment]
Yes, it is odd. I just opened another brand new Ledger S and loaded only EOS  app on it. 
I have 1.6.1 on the Ledger. 
I have not used the ledger with other EOS wallets. Just other coin wallets.

I disconnected the Ledger Live to see if that would help.  It allowed me not to be booted off the EOS in the ledger. But still endless Reinitializing in the Anchor. Then I turned on the Ledger Live again, and it booted me off again. 

Any other ideas? Do I need to find another EOS wallet that can be secured by Ledger? If yes, can you suggest one? Thanks Jay

**Comment by aaroncox** [comment]
1.3.1? Just confirming, I don't see a 1.6.1.

With Ledger Live turned off, and Anchor stuck on Reinitializing, try opening the Ledger section of the app and stopping it and then restarting it (or just restart Anchor with Ledger Live closed). See if that does the trick. 

If you're looking to try other wallets - you could try connecting it with Scatter (https://get-scatter.com/), but I have a feeling whatever is preventing the connection with Anchor will do the same with Scatter. Most EOSIO wallets use the same methods to connect with Ledger devices (HID) and have these same sorts of oddities.

**Comment by emmaivybriar** [comment]
Yes, it's 1.6.1 which is the latest version. 
I also updated my Ledger Live to the most recent update.
One thing I noticed is that in the Anchor screen to the right of "Ledger- Public Key"  it says "Load Public Key[0]"  I have let it stay at 0, which may be a mistake? I don't know how to find the Ledger Public Key. 
Is this affecting my results?

**Comment by aaroncox** [comment]
Very odd. I wonder if that version has something to do with the issue you're experiencing. In Ledger Live, all I see is 1.3.1:

![image](https://user-images.githubusercontent.com/677686/107909237-c7b60500-6f25-11eb-84b6-80f4be4a3add.png)

I'll have to do some digging to see why that is.

> One thing I noticed is that in the Anchor screen to the right of "Ledger- Public Key" it says "Load Public Key[0]" I have let it stay at 0, which may be a mistake?

That is correct, `0` is the first key derived by the Ledger and most people will just default to using it. You can change the index to any number you want - but for ease of use, typically you'll just want to leave it at 0.

When your Ledger is connected, the `Load Public Key[0]` button when clicked will load and display the first key returned from the Ledger. There will also be a "Verify on Device" button that when clicked will tell the Ledger to display the first key on the devices screen - which you can verify matches that which Anchor (or any other wallet) is reporting.

**Comment by emmaivybriar** [comment]
Actually the EOS is 1.3.1. I was referring to the Ledger Nano S as 1.6.1
Thanks for explaining the load public key option. 
If you happen to find out anything else about the ledger bug please let me know.

**Comment by aaroncox** [comment]
We'll keep looking and trying to reproduce the issue. I'll also be curious to know if Scatter is able to connect to your Ledger. If it's not, it's an issue with something on your computer - as opposed to something Anchor specific.

**Comment by robert-blue** [comment]
For those coming to this thread with the same issue.

I have found that when Ledger Live Manager is running then I get the same symptoms described by the OP. The EOS app on the ledger would just quit. Making sure Ledger Live Manager is not running allows me to use the ledger as expected.

TLDR; 
Don't run Ledger Live Manager when you want to use Anchor.
